flexmeasures/cli/tests/test_data_edit.py

```diff
@@ -5,12 +5,13 @@
 from sqlalchemy import select
 
 from flexmeasures.cli.tests.utils import to_flags
+from flexmeasures.data.models.audit_log import AssetAuditLog
 from flexmeasures.data.models.time_series import TimedBelief
 from flexmeasures.cli.tests.utils import get_click_commands
 from flexmeasures.tests.utils import get_test_sensor
 
 
-@pytest.mark.skip
+@pytest.mark.skip_github
```

Comment:
What informed this change?

Reply:
Testing new functionality I found that this test is not in fact ran while it works fine. I suggested that it was a typo / smth changed as I see no reason in having test in a codebase that is never run (it confuses as you see smth is being tested while it is in fact not)

Reply:
Ah you might be correct, thanks